### DISCUSSION TITLE
How do I use DBeaver to create the DLL (Data Definition Language) with indexes and foreign keys for all tables of a database in Informix?

### DISCUSSION THREAD

**Discussion by vi488234hd92** [dbeaver/dbeaver#21850] [Q&A]
I would like to create the DDL (Data Definition Language) with indexes, unique index and foreign keys for the Informix database for all tables in a certain database.

I have already found out how to create the DDL (Data Definition Language) for a certain number of tables in DBeaver. The problem here is that the CREATE statement, the index, the UNIQUE INDEX and the primary key are created, but the foreign key is not created.

Example:
```
CREATE TABLE settings (
	id serial NOT NULL,
	name char(50) NOT NULL,
	value char(256) NOT NULL,
	PRIMARY KEY (id) CONSTRAINT pk_settings_id
);

CREATE INDEX index_id ON settings (id);
CREATE UNIQUE INDEX index_id_name ON settings (id, name);
```

Do I need to activate something else or is it an error?

**Comment by LonwoLonwo** [comment]
Hello @viktorkuenstler 

Could you please try to push this button?

![2023-11-14 18_37_26-DBeaver  C__Users_anvolka_work_DbeaverP_dbeaver-workspace-idea  - C__Users_anvol](https://github.com/dbeaver/dbeaver/assets/45152336/f89bf876-b8e3-49ad-9590-fbb227af079f)

**Reply by vi488234hd92** [reply]
Hello @LonwoLonwo,

in my version this button "Separate FK" is not available in the side menu "DLL" of a database table.
I am using version 23.2.4 of the Community Edition.

In which DBeaver edition is this feature available?
